aeolus/model.cpp

```diff
@@ -963,7 +963,7 @@ bool Model::writePresets()
       uchar  data [256];
       FILE   *F;
 
-      sprintf (name, "%s/presets", _instr);
+      snprintf (name, sizeof(name), "%s/presets", _instr);
```

Comment:
`sizeof` though is an operator, IMHO there wqe want a space, actually here no parens at all are needed, `sizeof expr` (but `sizeof (type)`)